data-science-onramp/data-ingestion/setup.py

```diff
@@ -65,23 +85,39 @@ def convert_angle(angle):
                           weights=[0.55, 0.45])[0]
 
 
-def write_to_bigquery(df):
-    '''Write a dataframe to BigQuery'''
-
+def create_bigquery_dataset():
     # Create BigQuery Dataset
     client = bigquery.Client()
-    dataset_id = f'{client.project}.{RAW_DATASET_NAME}'
+    dataset_id = f'{client.project}.{DATASET_NAME}'
     dataset = bigquery.Dataset(dataset_id)
     dataset.location = "US"
     dataset = client.create_dataset(dataset)
 
+
+def write_to_bigquery(df, table_name):
+    '''Write a dataframe to BigQuery'''
+    client = bigquery.Client()
+    dataset_id = f'{client.project}.{DATASET_NAME}'
+
     # Saving the data to BigQuery
     df.write.format('bigquery') \
-        .option('table', f"{dataset_id}.{RAW_TABLE_NAME}") \
+        .option('table', f"{dataset_id}.{table_name}") \
         .option("temporaryGcsBucket", BUCKET_NAME) \
         .save()
 
-    print("Table successfully written to BigQuery")
+    print(f"Table {table_name} successfully written to BigQuery")
+
+
+def print_df(df, table_name):
+    '''Print 20 rows from dataframe and a random sample'''
+    # first 100 rows for smaller tables
+    df.show()
+
+    # random sample for larger tables
+    # for small tables this will be empty
+    df.sample(True, 0.0001).show(n=500, truncate=False)
+
+    print(f"Table {table_name} printed")
```

Comment:
Do you think printing the external datasets as well as the dirty one will affect some assert statements in our test script?

Reply:
As long as what your asserting is in `out` somewhere, I don't think it should matter